file_organizer_v2/src/file_organizer/services/deduplication/quality.py

```
@@ -0,0 +1,408 @@
+"""Image quality assessment for selecting best version among duplicates.
+
+This module provides quality scoring and comparison logic to automatically
+select the highest quality image from a group of similar/duplicate images.
+"""
+
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)